lib/ohai/plugins/linux/network.rb

```diff
@@ -108,6 +108,7 @@ def check_routing_table(family, iface, default_route_table)
           # http://rubular.com/r/pwTNp65VFf
           route_entry[k] = $1 if route_ending =~ /\b#{k}\s+([^\s]+)/
         end
+        route_entry["via"] = $1 if route_ending =~ /\bvia\s+inet6\s+([^\s]+)/
```

Comment:
can you add a rubular permalink like the one above for this regex so we can see exactly how it's matching?

Reply:
Don't the unittests show that? And even show them much clearer. I feel that's way more useful than a URL comment ... But I can add, will try work it out.